frontend/kesaseteli/employer/src/components/application/form/ActionButtons.sc.ts

```diff
@@ -1,12 +1,7 @@
+import FormSection from 'shared/components/forms/section/FormSection';
 import styled from 'styled-components';
 
-export const $ApplicationActions = styled.div`
-  display: flex;
-  justify-content: space-between;
-  margin-top: ${(props) => props.theme.spacing.l};
-`;
-
-export const $ApplicationAction = styled.div`
-  display: flex;
-  flex-direction: column;
+export const $ButtonSection = styled(FormSection)`
+  margin-top: ${(props) => props.theme.spacing.m};
+  padding-bottom: ${(props) => props.theme.spacing.m};
```

Comment:
IMHO, we may use the `css` prop for specifying margin values instead of creating a new styled component.

```TSX
 const theme = useTheme();

  <FormSection
    css={`
      margin-top: ${theme.spacing.m};
    `}>
     ...
```

Reply:
afaik, `styled-components` doesn't treat these as inline styles, but it will create a dedicated component under the hood.

Reply:
And margins specifically, I like to leave up to the parent component to determine its value.

Reply:
I dont know... I would rather keep the styles in sc.ts files and components more of the business logic and the dom structure. It's kinda tought that if I want to concentrate on business logic I work with components and hook. And when I want to fiddle with styles, then I look into sc.ts files.

Reply:
Fair enough!